src/shapes/Text.ts

```diff
@@ -201,12 +207,15 @@ export class Text extends Shape<TextConfig> {
     var lineTranslateX = 0;
     var lineTranslateY = 0;
 
+    context.setAttr('direction', direction);
```

Comment:
Need to think a bit about this line. Probably, we can skip it if the direction is `ltr`. Otherwise, we may need to fix many tests.